src/lib/accessibility/index.ts

```diff
@@ -12,3 +12,6 @@ export { default as InputBehavior } from './Behaviors/Input/InputBehavior'
 export { default as TreeBehavior } from './Behaviors/Tree/TreeBehavior'
 export { default as TreeItemBehavior } from './Behaviors/Tree/TreeItemBehavior'
 export { default as GroupBehavior } from './Behaviors/Tree/GroupBehavior'
+export { default as TabBehavior } from './Behaviors/Tab/TabBehavior'
+export { default as TabListBehavior } from './Behaviors/Tab/TabListBehavior'
+export { default as ToolbarBehavior } from './Behaviors/Toolbar/ToolbarBehavior'
```

Comment:
do we also need a ToolbarButton behavior? Menu.Item renders a link with ``role='menuitem'``. Can you please test with screen readers as well?

Reply:
Aria does not specify any custom role for the menu items themselves when it comes to toolbar. only on the container itself.

However in the example https://www.w3.org/TR/wai-aria-practices/examples/toolbar/toolbar.html they use Buttons as menu items, and we use links by default. It is the reason I used <Button> component as child and it was fine, <li> then <button>.
But now, with latest master, I have an anchor tag that has a button as child. So it's <li> <a> <button> which means double focus. And it's kind of incorrect as far as semantics are concerned.
I've tried a trick, and replaced Buttons with Icons, and it works. Icons now are acting as anchors. Should we leave it like this?

Another way is to override the anchor tag from the rendering MenuItem altogether and replace it with the button. Don't know if it's possible right now. I can check. What do you think?